net/dnscrypt-proxy/files/dnscrypt-proxy.init

```diff
@@ -33,7 +33,7 @@ create_config_file() {
     config_get      client_key      $1 'client_key'     ''
     config_get      syslog_prefix   $1 'syslog_prefix'  'dnscrypt-proxy'
     config_get      query_log_file  $1 'query_log_file' ''
-    config_get      log_level       $1 'log_level'	''
+    config_get      log_level       $1 'log_level'	'5'
```

Comment:
Hmm, I don't know.  That is odd that it says that.  My only guess is that either a) it's a typo or b) because they have LogLevel set to 6 in that config file (even though it's commented out).  My guess is it's simply a typo.